cmd/root.go

```diff
@@ -31,5 +42,19 @@ func Execute() int {
 	rootCmd.AddCommand(crawlCmd())
 	rootCmd.AddCommand(identifyCmd())
 	rootCmd.AddCommand(compareCmd())
+	rootCmd.AddCommand(dockerCmd())
+	rootCmd.PersistentFlags().StringSliceVar(&ignoreDirs, "ignore-dir", nil, `Specify directory pattern to ignore. Use multiple times to supply multiple patterns.
+Patterns should be relative to the provided root.
+e.g. ignore "^/proc" to ignore "/proc" when using a crawl root of "/"`)
+	rootCmd.PersistentFlags().DurationVar(&perArchiveTimeout, "per-archive-timeout", 15*time.Minute, `If this duration is exceeded when inspecting an archive, an error will be logged and the crawler will move onto the next file.`)
+	rootCmd.PersistentFlags().UintVar(&nestedArchiveMaxSize, "nested-archive-max-size", 5*1024*1024, `The maximum compressed size in bytes of any nested archive that will be unarchived for inspection.
```

Comment:
Does it make sense to have these as top-level flags when there are commands like `identify` and `compare` that presumably don't use all of these?

Might make more sense to factor out a function that adds flags for a crawl command and then apply that to the `dockerCmd` and `crawlCmd` (so that they have the same set of base flags but are not global).